boa/integrations/jupyter/jupyter.js

```diff
@@ -83,6 +83,12 @@
 
     /** Call the backend when the given function is called, handling errors */
     const handleCallback = func => async (token, ...args) => {
+        if (!colab) {
+            // Check if the cell was already executed. In Colab, eval_js() doesn't replay.
+            const response = await fetch(`../titanoboa_jupyterlab/callback/${token}`);
+            if (!response.ok) return;
```

Comment:
comment explaining the meaning of response.ok would be nice --
```suggestion
            // response.ok indicates the cell has already been executed
            if (!response.ok) return;
```

Reply:
comment explaining the meaning of response.ok would be nice --
```suggestion
            // server raises a 404 when the token was used before
            if (!response.ok) return;
```